generator/src/routes/+page.svelte

```diff
@@ -98,7 +98,8 @@
     // "Signed" and "Create valid signature" == "signed"
     // "Signed" and not "Create valid signature" == "invalid"
     // "Unsigned" == "url"
-    const security: "signed" | "invalid" | "url" = "signed"
+    const security: "signed" | "invalid" | "url" =
+      signOption === SignOption.SIGNED ? (isValid ? "signed" : "invalid") : "url"
```

Comment:
```suggestion
      signOption === SignOption.SIGNED ? (isValid ? "signed" : "invalid") : "unsigned"
```
At least task says so. Current file names: 
![Screenshot from 2023-11-21 17-29-53](https://github.com/ioxiocom/ioxio-tags/assets/73995378/75f800cb-9dbf-4896-974c-9aad8add50e4)